README.md

```diff
@@ -71,8 +71,9 @@ dotnet publish -c Release
 ##### You have supported hardware and drivers and [NVIDIA-Docker].
 
 ```
-git clone https://github.com/NVIDIA/model-analyzer
-docker build -f Dockerfile -t model-analyzer .
+git clone https://github.com/triton-inference-server/model_analyzer
+cd model_analyzer
+docker build -t model-analyzer .
```

Comment:
should be `docker build -t memory-analyzer`

Reply:
changed